kong/db/schema/entities/cluster_status.lua

```diff
@@ -0,0 +1,28 @@
+local typedefs      = require "kong.db.schema.typedefs"
+local ngx_time      = ngx.time
+
+return {
+  name        = "cluster_status",
+  primary_key = { "id" },
+
+  fields = {
+    { id = typedefs.uuid { required = true, }, },
+    { last_seen = typedefs.auto_timestamp_s },
+    { ip = typedefs.ip { required = true, } },
+    { config_hash = { type = "string", len_eq = 32, } },
+    { hostname = typedefs.host { required = true, } },
+  },
+
+  transformations = {
+    {
+      input = { "last_seen" },
+      on_read = function(last_seen)
+        if ngx_time() - last_seen > 60 then
```

Comment:
I would recommend to avoid adding the "status" field at this point as this is an API around which we do not have enough operational feedback. The consumer of the API can interpret the last_seen time as they see fit.

If we do go forward with this addition, please define `60` as a constant.

Reply:
This was a recommendation from the ADR, personally I am okay with removing it for now as it does not have practical use cases yet. We can always add it back in the future.